apps/glusterfs/operations_block_volume.go

```diff
@@ -86,8 +88,11 @@ func (bvc *BlockVolumeCreateOperation) Build() error {
 		}
 		reducedSize := ReduceRawSize(BlockHostingVolumeSize)
 		if len(volumes) > 0 {
-			bvc.bvol.Info.BlockHostingVolume = volumes[0].Info.Id
-			bvc.bvol.Info.Cluster = volumes[0].Info.Cluster
+			randSource := rand.NewSource(time.Now().UnixNano())
```

Comment:
Heketi's main.go seeds the global math rng (see https://github.com/heketi/heketi/blob/master/main.go#L480). There's no need to do it here esp. since this doesn't need much more than a simple shuffle.

Reply:
Thanks for pointing that out.